aws-replicator/aws_replicator/extension.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+from localstack.extensions.api import Extension, http
+from localstack.services.internal import get_internal_apis
+
+from aws_replicator.config import HANDLER_PATH
+from aws_replicator.server import RequestHandler
```

Comment:
these imports will be resolved when the entry point is resolved, i.e., when localstack starts. it would be better to defer the imports to the `update_gateway_routes` method.